packages/reassure-cli/src/commands/measure.ts

```diff
@@ -39,23 +44,33 @@ export async function run(options: MeasureOptions) {
   const defaultPath = process.platform === 'win32' ? 'node_modules/jest/bin/jest' : 'node_modules/.bin/jest';
   const testRunnerPath = process.env.TEST_RUNNER_PATH ?? defaultPath;
 
-  const defaultArgs = '--runInBand --testMatch "<rootDir>/**/*.perf-test.[jt]s?(x)"';
+  const defaultArgs = `--runInBand --testMatch "<rootDir>/**/*.perf-test.[jt]s?(x)" ${
+    options.silent ? '--silent' : ''
+  }`;
   const testRunnerArgs = process.env.TEST_RUNNER_ARGS ?? defaultArgs;
 
-  const spawnInfo = spawnSync(
-    'node',
-    [
-      '--jitless',
-      '--expose-gc',
-      '--no-concurrent-sweeping',
-      '--max-old-space-size=4096',
-      testRunnerPath,
-      testRunnerArgs,
-    ],
-    { shell: true, stdio: 'inherit', env: { ...process.env, OUTPUT_FILE: outputFile } }
-  );
-
-  console.log('');
+  const nodeArgs = [
+    '--jitless',
+    '--expose-gc',
+    '--no-concurrent-sweeping',
+    '--max-old-space-size=4096',
+    testRunnerPath,
+    testRunnerArgs,
+  ];
+  logger.verbose('Running tests: node ', nodeArgs.join(' '));
+
+  const spawnInfo = spawnSync('node', nodeArgs, {
+    shell: true,
+    stdio: 'inherit',
+    env: {
+      ...process.env,
+      OUTPUT_FILE: outputFile,
+      REASSURE_SILENT: options.silent.toString(),
+      REASSURE_VERBOSE: options.verbose.toString(),
+    },
+  });
+
+  logger.newLine();
 
   if (spawnInfo.status !== 0) {
     console.error(`❌  Test runner (${testRunnerPath}) exited with error code ${spawnInfo.status}`);
```

Comment:
Don't we want to `logger.error()` here as well? We don't necessary need anything from the new spawned process, do we.

Reply:
fixed here and in all other instances, except reassure-danger package as this one is strictly CLI and adding colors does not seem to make sense there.